js/libs/interface.js

```diff
@@ -0,0 +1,242 @@
+const widgetId = parseInt(Fliplet.Widget.getDefaultId(), 10)
+const widgetData = Fliplet.Widget.getData(widgetId) || {}
+
+import * as Sortable from 'sortablejs/Sortable.js'
+import flFloorplanColumns from '../config/default-table'
+
+Vue.directive('sortable', {
+  inserted(el, binding) {
+    if (Sortable) {
+      new Sortable(el, binding.value || {})
+    }
+  }
+});
+
+const selector = '#flooplan-app'
+
+const app = new Vue({
+  el: selector,
+  data() {
+    return {
+      appName: Fliplet.Env.get('appName'),
+      organizationId: Fliplet.Env.get('organizationId'),
+      defaultColumns: flFloorplanColumns,
+      autoDataSource: widgetData.autoDataSource || false,
+      dataSources: [],
+      filePickerProvider: null,
+      floorPanelsIsEmpty: true,
+      settings: widgetData,
+      floors: widgetData.floors || [],
+      markers: widgetData.markers || [],
+      hasError: false,
+      showAddMarkersUI: false
+    }
+  },
+  methods: {
+    makeid(length) {
```

Comment:
why not using `Fliplet.guid()`?

Reply:
To freaking big the number returned by it.

Reply:
Please elaborate on what you use this number for so maybe I can suggest.

Reply:
@squallstar I use it to create IDs for each floor and for each marker style, those ids are saved in the config but also shown in the DOM.

Reply:
@hcarneiro I would recommend you use `Fliplet.guid()` :)

Reply:
Fiiiiiineee :D